### ISSUE TITLE
NVDA fails to read popup notifications from Windows Game Bar

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#18910]
### Steps to reproduce:
1. Make sure that Windows **Game Bar** is installed on your system and it is up-to-date (update it via the **Microsoft Store** if necessary).
2. Go to the desktop.
3. Press **Windows+Alt+Print Screen** to make a screenshot via the Game Bar.
4. Verify that a notification from **Game Bar** is displayed in the middle right side of the screen, informing of the successfully made screenshot
5. Verify that NVDA reads the content of the notification from the **Game Bar**.

### Actual behavior:
- The notification is displayed.
- NVDA does not read the notification.

### Expected behavior:
- The notification is displayed.
- NVDA does read the notification.

### NVDA logs, crash dumps and other attachments:
Not applicable.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
alpha-52562,3d74061b (2026.1.0.52562)

#### Windows version:
Windows 11 Pro version 24H2 (build 26100.6584), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".

#### Name and version of other software in use when reproducing the issue:
Windows Game Bar version: 7.325.7090.0

### Notes:
- The issue is mostly noticeable when playing a game like **Forza Motorsport 2023** and an achievement from the game is unlocked - then a notification from **Game Bar** is displayed about the achievement and NVDA does not read it.
- I'm not sure if the issue occurs with **Narrator** or not - it seams to read those notifications, but I'm not sure if it is on purpose or by accident just because the notification sometimes gets focused.

#### Does the issue still occur after restarting your computer?
Yes, it does.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes - 2025.2 and the problem seams to be the same.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes, it is.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not applicable.
